EC-Switch-Seclog.py

```diff
@@ -0,0 +1,973 @@
+            else:
+                tseclog_profile = arg
+        if opt in ("-o", "--org"):
+            if not arg:
+                print(f"Organisation profile has not been provided. [{Status.FAIL.value}]")
+                usage()
+                sys.exit(1)
+            else:
+                organisation_profile = arg
+        
+    
+    logging.basicConfig(level=verbosity)
+
+    # linked account
+    
+    print(f"Account to be moved : {account_profile}")
+    account_session=boto3.Session(profile_name=account_profile, region_name ='eu-west-1')
+    
+    account_id = get_account_id(account_session)
+
+    print(f"Account ID : {account_id}")
+    
+    # get stored seclog_id from ssm parameter
+    ssm = account_session.client('ssm')
+    try:
+        response = ssm.get_parameter(Name='/org/member/SecLogMasterAccountId')
+        stored_sseclog_id = response['Parameter']['Value']
+    except ClientError as err:
+        print(f"LZ not configured on this account. [{Status.FAIL.value}]")
+        print("Exiting...")
+        sys.exit(1)
+
+    try:
+        response = ssm.get_parameter(Name='/org/member/SecLog_sns_arn')
+        stored_SecLog_sns_arn = response['Parameter']['Value']
+    except ClientError as err:
+        print(f"SecLog_sns_arn not configured on this account. [{Status.FAIL.value}]")
+        print("Exiting...")
+        sys.exit(1)
+
+    try:
+        response = ssm.get_parameter(Name='/org/member/KMSCloudtrailKey_arn')
+        stored_KMSCloudtrailKey_arn = response['Parameter']['Value']
+    except ClientError as err:
+        print(f"KMSCloudtrailKey_arn not configured on this account. [{Status.FAIL.value}]")
+        print("Exiting...")
+        sys.exit(1)
+
+    try:
+        response = ssm.get_parameter(Name='/org/member/SecLogOU')
+        stored_SecLogOU = response['Parameter']['Value']
+    except ClientError as err:
+        print(f"SecLogOU not configured on this account. [{Status.FAIL.value}]")
+        print("Exiting...")
+        sys.exit(1)
+
+    print(f"Linked account identified. [{Status.OK.value}]")
+    print("")
+
+    # organisation account
+
+    print(f"Organisation account : {organisation_profile}")
+    
+    org_session=boto3.Session(profile_name=organisation_profile, region_name ='eu-west-1')
+
+    organisation_id = get_account_id(org_session)
+    organizations = org_session.client('organizations')
+    response = organizations.describe_account(AccountId=account_id)
+    account_email = response['Account']['Email']
+
+    print(f"Account ID : {organisation_id}")
+    print(f"Organisation account identified. [{Status.OK.value}]")
+    print("")
+
+    # original seclog account
+    print(f"Original SECLOG account : {sseclog_profile}")
+   
+    sseclog_session=boto3.Session(profile_name=sseclog_profile, region_name ='eu-west-1')
+
+    sseclog_id = get_account_id(sseclog_session)
+    print(f"Account ID : {sseclog_id}")
+    if (is_seclog(sseclog_session) == False):
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The `==` and `!=` operators use the compared objects&#39; ` __eq__` method to test if they are equal. To check if an object is a singleton, such as `None`, we recommend that you use the `is` identity comparison operator.

[Learn more](https://docs.python.org/3/reference/expressions.html#is)

Similar issue at line number 197.